.ci_support/linux_c_compilergcccxx_compilergxxfortran_compilergfortran.yaml

```diff
@@ -0,0 +1,22 @@
+build_number_decrement:
+- '0'
+c_compiler:
+- gcc
+channel_sources:
+- conda-forge/label/gcc7,defaults
```

Comment:
Seems like the `conda-forge` channel is missing? It's not finding the `mpi` metapackage, which is in the conda-forge channel.

Reply:
Hmm, ok.

Reply:
@conda-forge/core We might need to copy some recipes over to the new channel to get things to work properly. Thoughts?

Reply:
We didn't want to risk the inclusion of things from `main`. If for instance we missed a dependency we wanted the build to fail outright rather than run the risk of incorrectly linking something.

Reply:
Can you link me to the conversation?

Reply:
> We might need to copy some recipes over to the new channel to get things to work properly

It's not "some". It's a lot of recipes. Things like `setuptools, pytest` are some major ones, but you'll have to copy all the packages without compiled objects to the new channel for this to work.

Reply:
I think that conversation was in person with @mariusvniekerk but it is also discussed in the [CFEP](https://github.com/conda-forge/conda-forge-enhancement-proposals/pull/10)